tests/utils/helpers.rs

```diff
@@ -449,6 +449,11 @@ pub fn uda_token_data(
 }
 
 impl TestWallet {
+
+    pub fn raw_rgb_wallet(&self) -> &RgbWallet<Wallet<XpubDerivable, RgbDescr>> {
+        &self.wallet
+    }
+    
```

Comment:
please instead of exposing the wallet add a `fungible_history` pub wrapper method here (better after the `contract_data_allocations` method)